test/test_manifest.json

```diff
@@ -10482,5 +10482,13 @@
     "rounds": 1,
     "link": true,
     "type": "eq"
+  },
+  {
+    "id": "bug1919513",
+    "file": "pdfs/bug1919513.pdf",
+    "md5": "83d1a7fab2c81e632910c334879e7334",
+    "rounds": 1,
+    "talos": false,
```

Comment:
Any reason for this line?

Reply:
Yes, the people looking at the results are bit overwhelmed and they'd like to have less pdfs.
So we should try to reduce the number of them we've in Talos.
With this specific pdf, I don't think we'll have something significant to measure, so it's why I removed it from the list.
I'll try to take some time to check all of them and decide the pdfs we want to skip.